API.md

```diff
@@ -31,4 +31,4 @@ Parameter name                | Value   | Description
 api_token                     | string  | A token that identifies a unique user
 prediction[description]       | string  | Your prediction statement
 prediction[deadline]          | date    | When it will/won't have happened
-prediction[private]           | boolean | True if you want it to be private (optional)
+prediction[visibility]        | string  | visible_to_everyone (default), visible_to_creator
```

Comment:
It is actually backward compatible. My first commit broke the API but a later commit went back and verified (and tested) that both old and new sets of attributes are supported.